src/core/agent/services/multiSigService.ts

```diff
@@ -693,6 +694,17 @@ class MultiSigService extends AgentService {
       .exchanges()
       .send(name, "multisig", aid, route, payload, embeds, recp);
   }
+
+  @OnlineOnly
+  async hasMultisig(multisigId: string): Promise<boolean> {
+    const multiSig = await this.identifierStorage
+      .getIdentifierMetadata(multisigId)
+      .catch(() => undefined);
```

Comment:
I think we should re-throw here if it's not `IdentifierStorage.IDENTIFIER_METADATA_RECORD_MISSING` in case something else went wrong